docs/importing.md

```diff
@@ -0,0 +1,73 @@
+# Importing existing Data into Chronicle
+
+## Importing Data into Chronicle
+
+Chronicle provides a CLI command `import` to load data from a JSON-LD file that
+contains an array of Chronicle Operations. These operations can be used to
+create agents, namespaces, and other resources in the Chronicle database. For
+more information about the data format of the import file, see [Recording Provenance](../recording_provenance/#importing-data-into-chronicle).
+
+## Example import process
+
+### Local development environment
+
+To import data into Chronicle, follow these steps:
+
+- Create a JSON-LD file with an array of Chronicle Operations.
+- Save the file in the Chronicle root directory with a unique name, for example,
+  `import.json`.
+- Run the following command:
+
+  ```bash
+  chronicle --bin chronicle -- import <namespace-id> <namespace-uuid> <json-file-url>
+  ```
+
+See the [CLI `import` command help](../recording_provenance/#importing-data-into-chronicle)
+for more information on each argument in the command.
+
+If the data is successfully imported, the Chronicle Operations will be added
+to the Chronicle database.
+
+### Docker compose environment
+
+Note: This example assumes you are using the chronicle-examples and running the
+stl version of the docker compose file. You will need to adapt the instructions
+to your environment, specifically the name of the chronicle container.
+
+To import data into Chronicle, follow these steps:
+
+- Create a JSON-LD file with an array of Chronicle Operations.
+- Add a bind mount to your docker compose file mounting your file in the
+  chronicle container. If you're using the [chronicle-examples](https://github.com/btpworks/chronicle-examples/)
+  or [chronicle-bootstrap](https://github.com/btpworks/chronicle-bootstrap)
+  configuration then this will be for the `chronicle-sawtooth-api` container,
+  as defined in  `docker/chronicle.yaml`.
+
+  ```yaml
+  volumes:
+    - type: bind
+      source: ~/import.json
```

Comment:
```suggestion
      source: ./import.json
```

I was just working on adding support for `import` to examples and caught that earlier you say, "Save the file in the Chronicle root directory," but here it's saved to `~/`. 

It might be better to update `chronicle-examples` with an empty `import.json` file in the root of the `chronicle-examples` repo like I've done in [this branch](https://github.com/btpworks/chronicle-examples/compare/main...feature/make-import-work-with-examples). That way we can just add the bind mount to the source code.

Reply:
I don't think we need the tracked blank file, the instructions have us place a file before starting docker. Still, I'd agree that the instructions *do* need a bit of fixing here to ensure that the `import.json` is placed where the compose file expects, well-spotted.

Reply:
Makes sense to clarify it more, I've updated the text and added note about changing it for your own environment. What do you think?